ssz/__init__.py

```diff
@@ -1,6 +1,9 @@
 #
 # sedes
 #
+from . import (  # noqa: F401,
+    tools,
+)
```

Comment:
I don't think this is ok.  The idea behind the `library.tools` abstraction is that the main library code never imports it because it might have dependencies that are not installed by default.  We can keep this **if** we have a CI test that does a naive install of the library (without any testing or development dependencies) and verifies that `import ssz` does not error out.

Reply:
Thanks for the review. I was not aware that it reintroduced the dependency back and didn't realize `ssz.tools.to_formatted_dict` is a bad import pattern. Give me a 👍and I'll revert this PR.